### PR TITLE
Genetics genes expansion

### PR DESCRIPTION
**Pull request by LongSleeves**
Fixes surgery overlays not being removed
Makes use of slime bones sense they are just sitting there in code.
Makes slimes processable by the genetics machines.
Adds eyes to the flesh sacs. Cause my nightmare fuel isn't good enough.
Updates the genetics posters with more combinations
Adds a testing bioshock thats meant to only have the active genes we are testing during development

Adds 2 new mutations
-Spark breath
-Flame retardant

Thanks to @Sebastian Schrader and @I The Magician (Hex) for the grammer checks and suggestions.

### CODE REVIEW

File: code/modules/genetics/machinery/genetics_items.dm [hunk 1]
```diff
@@ -560,7 +560,12 @@ Neglects to mention where to find its pieces.
 		"Roach DNA can be irradiated into a plethora of different breeds!",
 		"Warning- The Gigantism gene combined with roach DNA can have hazardous results!",
 		"Spider DNA can be irradiated into a plethora of different breeds!",
-		"The Gigantism gene only produces an Empress spider when combined with egg-laying spider DNA",
+		"Termite DNA can be irradiated to a plethora of different breeds!",
+		"Some termites can be made even larger!",
+		"Loud sounds mixed with specific friends can lead to buddy's when you need it most.",
+		"Cats can be made both bigger and smaller!",
+		"Making a dog to big will result in a bear.",
```
Comment (ThePainkiller) [inline code/modules/genetics/machinery/genetics_items.dm:567]:
```suggestion
		"Making a dog too big will result in a bear.",
```

File: code/modules/genetics/mutations/flame_retardent.dm [hunk 2]
```diff
@@ -0,0 +1,23 @@
+//Macro: MUTATION_FLAME_RETARDENT
+//runs a process every now and then to extinguish the holder
+/datum/genetics/mutation/flame_retardent
+	name = "Flame Retardent"
```
Comment (ThePainkiller) [inline code/modules/genetics/mutations/flame_retardent.dm:4]:
```suggestion
	name = "Flame Retardant"
```

File: code/modules/genetics/mutations/flame_retardent.dm [hunk 3]
```diff
@@ -0,0 +1,23 @@
+//Macro: MUTATION_FLAME_RETARDENT
+//runs a process every now and then to extinguish the holder
+/datum/genetics/mutation/flame_retardent
```
Comment (ThePainkiller) [inline code/modules/genetics/mutations/flame_retardent.dm:3]:
```suggestion
/datum/genetics/mutation/flame_retardant
```

File: code/modules/genetics/mutations/flame_retardent.dm [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	name = "Flame Retardent"
+	key = "MUTATION_FLAME_RETARDENT"
```
Comment (ThePainkiller) [inline code/modules/genetics/mutations/flame_retardent.dm:5]:
```suggestion
	key = "MUTATION_FLAME_RETARDANT"
```

File: code/modules/genetics/mutations/flame_retardent.dm [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+	desc = "Causes a flame smothering fluid to be produced on the skin."
```
Comment (ThePainkiller) [inline code/modules/genetics/mutations/flame_retardent.dm:6]:
```suggestion
	desc = "Causes a flame-smothering fluid to be permanently produced by the skin."
```

File: code/modules/genetics/mutations/flame_retardent.dm [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+	gain_text = "Your skin feels hydrated."
+	instability = 10
+	exclusive_type = MUT_TYPE_SKIN
+
+/datum/genetics/mutation/flame_retardent/onMobImplant()
```
Comment (ThePainkiller) [inline code/modules/genetics/mutations/flame_retardent.dm:11]:
```suggestion
/datum/genetics/mutation/flame_retardant/onMobImplant()
```

File: code/modules/genetics/mutations/flame_retardent.dm [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+	initializeProcessing()
+
+/datum/genetics/mutation/flame_retardent/onMobRemove()
```
Comment (ThePainkiller) [inline code/modules/genetics/mutations/flame_retardent.dm:14]:
```suggestion
/datum/genetics/mutation/flame_retardant/onMobRemove()
```

File: code/modules/genetics/mutations/flame_retardent.dm [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+	haltProcessing()
+
+/datum/genetics/mutation/flame_retardent/Process()
```
Comment (ThePainkiller) [inline code/modules/genetics/mutations/flame_retardent.dm:17]:
```suggestion
/datum/genetics/mutation/flame_retardant/Process()
```

File: code/modules/genetics/mutations/item_makers.dm [hunk 9]
```diff
@@ -146,3 +146,23 @@
...
+	name = "Humanoid eyes"
+	key = "MUTATION_EYES"
+	desc = "When cloned in an organ sac, produces functional eyes from an unidentified substance."
+	sac_desc_modifier = "It focuses on its surroundings."
+	loot = list(/obj/item/organ/internal/eyes)
+
+/datum/genetics/mutation/sac_contents/marqua_eyes
+	name = "Marqua eye"
+	key = "MUTATION_MARQUA_EYES"
+	desc = "When cloned in an organ sac, produces a single marqua like eye from an unidentified substance."
```
Comment (ThePainkiller) [inline code/modules/genetics/mutations/item_makers.dm:159]:
```suggestion
	desc = "When cloned in an organ sac, produces a single Mar'qua-like eye from an unidentified substance."
```

File: code/modules/genetics/mutations/item_makers.dm [hunk 10]
```diff
@@ -146,3 +146,23 @@
...
+	sac_desc_modifier = "It watches the world around it."
+	loot = list(/obj/item/organ/internal/eyes/marqua)
+
+/datum/genetics/mutation/sac_contents/slime_bone
+	name = "Slime bone"
+	key = "MUTATION_SLIME_BONE"
+	desc = "When cloned in an organ sac, creates a slimy cartalidge from an unidentified substance."
```
Comment (ThePainkiller) [inline code/modules/genetics/mutations/item_makers.dm:166]:
```suggestion
	desc = "When cloned in an organ sac, creates a slimy cartilage from an unidentified substance."
```

File: code/modules/genetics/mutations/spark_breath.dm [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+	//safty check to remove the verb if the mutation isn't there.
+	var/datum/genetics/mutation/spark_breath/spk = src.unnatural_mutations.getMutation("MUTATION_SPARK_BREATH", TRUE)
+	if(!spk)
+		src.verbs -= /mob/living/proc/mutation_spark_breath
+		return
+
+	//Check if we have a smokeable in our hand.
+	if (istype(usr.get_active_hand(), /obj/item/clothing/mask/smokable))
+		var/obj/item/clothing/mask/smokable/ciggy = usr.get_active_hand()
+		usr.visible_message("[usr] lights \the [ciggy] with his breath.")
```
Comment (ThePainkiller) [inline code/modules/genetics/mutations/spark_breath.dm:35]:
```suggestion
		usr.visible_message("[usr] lights \the [ciggy] with their breath.")
```

File: code/modules/genetics/recipes/abilities.dm [hunk 12]
```diff
@@ -3,3 +3,12 @@
 /datum/genetics/mutation_recipe/combination/wurm_cry
 	required_mutations = list(MUTATION_SCREAMING, MUTATION_TERMITE_FRIEND)
 	result_path = MUTATION_WURM_CRY
+
+/datum/genetics/mutation_recipe/combination/spark_breath
+	required_on_clone_types = list(/mob/living/carbon/superior_animal/giant_spider/hunter/shocker)
+	required_mutations = list(MUTATION_PSN_BREATH)
+	result_path = MUTATION_SPARK_BREATH
+
+/datum/genetics/mutation_recipe/combination/flame_retardent
```
Comment (ThePainkiller) [inline code/modules/genetics/recipes/abilities.dm:12]:
```suggestion
/datum/genetics/mutation_recipe/combination/flame_retardant
```

File: code/modules/genetics/recipes/abilities.dm [hunk 13]
```diff
@@ -3,3 +3,12 @@
...
+	required_mutations = list(MUTATION_COW_SKIN, MUTATION_IMBECILE)
+	result_path = MUTATION_FLAME_RETARDENT
```
Comment (ThePainkiller) [inline code/modules/genetics/recipes/abilities.dm:14]:
```suggestion
	result_path = MUTATION_FLAME_RETARDANT
```

File: code/modules/genetics/mutations/flame_retardent.dm [hunk 14]
```diff
@@ -0,0 +1,23 @@
+//Macro: MUTATION_FLAME_RETARDENT
```
Comment (ThePainkiller) [inline code/modules/genetics/mutations/flame_retardent.dm:1]:
```suggestion
//Macro: MUTATION_FLAME_RETARDANT
```
I also suggest changing the filename if you want, for the sake of consistency.

File: sojourn-station.dme [hunk 15]
```diff
@@ -1899,6 +1899,7 @@
 #include "code\modules\genetics\mutations\dwarfism.dm"
 #include "code\modules\genetics\mutations\echolocation.dm"
 #include "code\modules\genetics\mutations\emp_blast.dm"
+#include "code\modules\genetics\mutations\flame_retardent.dm"
```
Comment (ThePainkiller) [inline sojourn-station.dme:1902]: Ditto here, if you change the filename for consistency.

**Review by ThePainkiller — CHANGES_REQUESTED** [review]
Typo fixes and a request/suggestion

**Comment by Guidesu** [comment]
I recommend using src or I forgot the command name to actually use the person's gender instead of neutral

File: code/modules/genetics/mutations/flame_retardent.dm (hunk 14 above)
Comment (LongSleeves) [reply code/modules/genetics/mutations/flame_retardent.dm:1]: oh didn't see I spelled it diffrently in one spot. My bad. Good catch.

File: code/modules/genetics/mutations/spark_breath.dm (hunk 11 above)
Comment (LongSleeves) [reply code/modules/genetics/mutations/spark_breath.dm:35]: you can tell I was getting to the end of my coding push. Lots of spelling mistakes.

**Comment by LongSleeves** [comment]
Instead of hitting commit 50 times for suggestions I think from now I'll I'll just boot up the actual PR for edit. To many quick changes with various popups and updates.

**Pull request merged**